modules/users/manifests/samsimpson.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the samsimpson user
+class users::samsimpson {
+  govuk_user { 'samsimpson':
+    fullname => 'Sam Simpson',
+    email    => 'sam.simpson@digital.cabinet-office.gov.uk',
+    ssh_key  => 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHGtiWG65GGmpzqG48BWeB6TXTx8hdR26uu2MyG+JCMa sam.simpson@digital.cabinet.office.gov.uk',
```

Comment:
We're on a pretty ancient Ubuntu (14.04 ESM) but I *think* ed25519 should just about work 😅

Reply:
Oh I see there are already a few ed25519 keys - I think it's a pretty safe bet that our openssh is new enough.